docs/mddocs/Quickstart/llama_cpp_quickstart.md

```diff
@@ -234,112 +333,135 @@ llm_load_print_meta: f_norm_eps       = 0.0e+00
 
-#### Program crash with Chinese prompt
+#### 9. Program crash with Chinese prompt
 If you run the llama.cpp program on Windows and find that your program crashes or outputs abnormally when accepting Chinese prompts, you can open `Region->Administrative->Change System locale..`, check `Beta: Use Unicode UTF-8 for worldwide language support` option and then restart your computer.
 
 For detailed instructions on how to do this, see [this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469).
 
-#### sycl7.dll not found error
+#### 10. sycl7.dll not found error
 If you meet `System Error: sycl7.dll not found` on Windows or you meet similar error on Linux, please check:
 
 1. if you have installed conda and if you are in the right conda environment which has pip installed oneapi dependencies on Windows
 2. if you have executed `source /opt/intel/oneapi/setvars.sh` on Linux
 
-#### Check driver first when you meet garbage output
+#### 11. Check driver first when you meet garbage output
 If you meet garbage output, please check if your GPU driver version is >= [31.0.101.5522](https://www.intel.cn/content/www/cn/zh/download/785597/823163/intel-arc-iris-xe-graphics-windows.html). If not, please follow the instructions in [this section](./install_linux_gpu.md#install-gpu-driver) to update your GPU driver.
 
-#### Why my program can't find sycl device
-If you meet `GGML_ASSERT: C:/Users/Administrator/actions-runner/cpp-release/_work/llm.cpp/llm.cpp/llama-cpp-bigdl/ggml-sycl.cpp:18283: main_gpu_id<g_all_sycl_device_count` error or similar error, and you find nothing is output when using `ls-sycl-device`, this is because llama.cpp cannot find the sycl device. On some laptops, the installation of the ARC driver may lead to a forced installation of `OpenCL, OpenGL, and Vulkan Compatibility Pack` by Microsoft, which inadvertently blocks the system from locating sycl devices. This issue can be resolved by manually uninstalling it in Microsoft store.
\ No newline at end of file
+#### 12. Why my program can't find sycl device
+If you meet `GGML_ASSERT: C:/Users/Administrator/actions-runner/cpp-release/_work/llm.cpp/llm.cpp/llama-cpp-bigdl/ggml-sycl.cpp:18283: main_gpu_id<g_all_sycl_device_count` error or similar error, and you find nothing is output when using `ls-sycl-device`, this is because llama.cpp cannot find the sycl device. On some laptops, the installation of the ARC driver may lead to a forced installation of `OpenCL, OpenGL, and Vulkan Compatibility Pack` by Microsoft, which inadvertently blocks the system from locating sycl devices. This issue can be resolved by manually uninstalling it in Microsoft store.
+
+#### 13. Core dump when having both integrated and dedicated graphics
+If you have both integrated and dedicated graphics on your computer and didn't specify which device to use, it will cause a core dump. Therefore, you need to run `export ONEAPI_DEVICE_SELECTOR=level_zero:0` before running `llama-cli`.
```

Comment:
If you have both integrated and dedicated graphics displayed in your llama.cpp's device log and don't specify which device to use, it will cause a core dump. In such case, you may need to specify `export ONEAPI_DEVICE_SELECTOR=level_zero:0` before running `llama-cli`.